TwitchLib.Client/TwitchClient.cs

```diff
@@ -99,6 +91,10 @@ public class TwitchClient : ITwitchClient
         #endregion
 
         #region Public Variables
+        /// <inheritdoc/>
+        public ICollection<char> ChatCommandIdentifiers { get; } = new HashSet<char>();
+        /// <inheritdoc/>
+        public ICollection<char> WhisperCommandIdentifiers { get; } = new HashSet<char>();
```

Comment:
Previously it was adding `!` by default. Do we change that behavior?

Reply:
it depens, but I think we shoud.
initializing with a custom identifier will look more "natural"

Without `!`:
```cs
var client = new TwitchClient()
{
   ChatCommandIdentifiers = { '?' }
};
```
With `!`:
```cs
var client = new TwitchClient();
client.ChatCommandIdentifiers.Clear();
client.ChatCommandIdentifiers.Add('?');

// or

var client = new TwitchClient()
{
   ChatCommandIdentifiers = { '?' }
};
client.ChatCommandIdentifiers.Remove('!');
```

Reply:
I think having a default is preferred but I am sure it could be handled better internally. like if we have a non empty ChatCommandIdentifiers array coming in dont override it/add the '!' so we dont need the remove stuff

Reply:
solved, '!' is added in `Initialize` if no identifier was added before
(the alternative is `ConnectAsync` but I think `Initialize` is a better choice)